### ISSUE TITLE
Investiguer si les activités peuvent permettre de faire une frise

### ISSUE THREAD

**Issue by agarrone** [etalab/data.gouv.fr#1005]
Voir notamment ce qu'on peut afficher sur ce qui est moissonné 

![Image](https://user-images.githubusercontent.com/60264344/208107899-1e75e656-7c5a-4933-8d04-9e5a84c90583.png)

**Comment by quaxsze** [comment]
L'idée de frise a été tentée par le passé puis abandonnée. On s'était rendu compte que c'était une fausse bonne idée pour au moins deux raisons :
- Premièrement la base de donnée activité permet de savoir qui a modifié quoi et quand mais pas ce qui a été modifié.
- Deuxièmement la majorité des JDD étant moissonnés, l'activité est inexistante.

PS: La notation "il y a 3 jours" est un enfer 😬

**Comment by agarrone** [comment]
Thanks. 

- On a le "qui" ? C'est trop bien c'est demandé par les producteurs à voir comment on interface ça avec la métadonnées point de contact @maudetes 
- Comment pourrait savoir ce qui a été modifié ? 
- Sur le moissonage on a que date de création et dernier job ? On sait si y'a eu une mise à jour sur le dernier job non ? On a pas l'historique par contre c'est ça ?

**Comment by maudetes** [comment]
> On a le "qui" ? C'est trop bien c'est demandé par les producteurs à voir comment on interface ça avec la métadonnées point de contact @maudetes

Yes, ça peut être intéressant pour notre point de contact! A garder en tête que par contre un point de contact vers un individu c'est risqué, on pourrait plutôt préférer des mails de contact génériques plus durables (et moissonnable, exposable, ce qu'on ne veut pas avec un contact individuel).

**Comment by maudetes** [comment]
Pas prioritaire pour le moment.
